surfer/viz.py

```diff
@@ -2652,7 +2652,7 @@ def save_movie(self, fname, time_dilation=4., tmin=None, tmax=None,
             raise ValueError("tmax=%r is greater than the latest time point "
                              "(%r)" % (tmax, self._times[-1]))
         n_frames = floor((tmax - tmin) * time_dilation * framerate)
-        times = np.arange(n_frames)
+        times = np.arange(n_frames).astype(float)
```

Comment:
Probably not a big performance hit here, but you could pass the dtype to arange to avoid making a copy.